#4431 DataGrid's Column extension: Added ability to change on first click sort direction

injDakov opened the pull request:
This is a simple improvement with a small impact on the existing functions.

This improvement adds the property FirstClickSortDirection gives the ability of the developer to select which will be sort direction on the first click, instead of always being ASC and after that, on the second click be DESC.

[Feature #4431](https://github.com/Megabit/Blazorise/issues/4431)

David-Moreira commented:
> `*Click*` feels strange. I think naming it `bool ReverseSorting` will make more sense to the users.
> 
> @David-Moreira what do you think?
> 
> PS. @injDakov can you please accept the CLA?

I agree, since this does not affect only click, but sorting in general. I believe programatically calling the Sort method will have the same effect right?

I like the new naming, `ReverseSorting` or use the same naming as in the new bool flag, `InverseSorting`. 
Eitherway keep the flag and the new Parameter naming consistent.

Also this line should read better in the reverse order
`return !isInverse ? SortDirection.Ascending : SortDirection.Descending;`
**Prefer** 
`return isInverse ? SortDirection.Descending : SortDirection.Ascending;`

injDakov commented:
I have read the CLA Document and I hereby sign the CLA

injDakov commented:
recheck

injDakov commented:
@stsrki  @David-Moreira  

I implemented the requested changes.
But I am not sure how to accept the CLA, I added comments "I have read the CLA Document and I hereby sign the CLA" and "recheck"  but looks is still not OK. 

Also see that have a mess with my local GIT settings, but hope that this is not related to CLA issue.

David-Moreira commented:
Hello,

Unfortunately it seems you didn't add proper metadata to the commits. Could you ammend these? It's probably easier to submit a new PR if you don't mind. Your user and email should be configured when submitting a commit.
![image](https://user-images.githubusercontent.com/22283161/210585629-f5761867-0843-4c47-9544-13a4d9c9a453.png)
